.github/workflows/pr_and_main.yml

```diff
@@ -22,9 +22,6 @@ jobs:
       - run: python -m pip install --upgrade pip
       - run: python -m pip install -r requirements/ci.txt
       - run: tox -e static
-      - uses: stefanzweifel/git-auto-commit-action@v5
```

Comment:
We don't use the stefanzweifel action, but we use something else instead to apply the formatting I think?
https://github.com/scipp/copier_template/blob/main/template/.github/workflows/ci.yml#L30

Reply:
Correct